src/config.rs

```diff
@@ -293,6 +293,10 @@ pub struct Config {
 
     // If true, when AppTunnel is set for
     pub localhost_app_tunnel: bool,
+
+    pub ztunnel_identity: identity::Identity,
+
+    pub ztunnel_workload: state::WorkloadInfo,
```

Comment:
These should probably be options. If someone doesn't set the variables ztunnel shouldn't fail, it just won't be able to start its own inbound listener and proxy to itself.

Reply:
or at the very least, it should be optional in Dedicated mode, which would never do this and should not be required to set the env vars (they typically are run outside of k8s in this mode)